cloud/linode/common.go

```diff
@@ -42,3 +44,14 @@ func IgnoreLinodeAPIError(err error, code int) error {
 
 	return err
 }
+
+func isPrivate(ip *net.IP) bool {
+	if Options.LinodeExternalNetwork == nil {
+		return ip.IsPrivate()
+	}
+	ipAddr, err := netip.ParseAddr(ip.String())
```

Comment:
Why are we re-parsing the IP only to check it belong to a subnet? Can't we define `LinodeExternalNetwork` as `net.IPNet` and use [`IPNet.Contains`](https://pkg.go.dev/net#IPNet.Contains)?

Reply:
updated, thanks